cmd/init.go

```diff
@@ -54,126 +72,144 @@ to quickly create a Cobra application.`,
 		if err != nil {
 			return err
 		}
-
-		log.Println("github:", githubFlags.GithubHost)
-		log.Println("dry run enabled:", globalFlags.DryRun)
-
-		if len(awsFlags.AssumeRole) > 0 {
-			log.Println("calling assume role")
-			err := aws.AssumeRole(awsFlags.AssumeRole)
-			if err != nil {
-				log.Println(err)
-				return err
+		go func() {
+
+			log.Println("github:", githubFlags.GithubHost)
+			log.Println("dry run enabled:", globalFlags.DryRun)
+
+			if len(awsFlags.AssumeRole) > 0 {
+				log.Println("calling assume role")
+				err := aws.AssumeRole(awsFlags.AssumeRole)
+				if err != nil {
+					log.Println(err)
+					//return err
+				}
+				log.Printf("assuming new AWS credentials based on role %q", awsFlags.AssumeRole)
 			}
```

Comment:
I believe for safety, this step should be kept serial.. as I believe the bucket creation step will use this information to create a bucket with the role in play.